core/src/main/java/nl/nn/adapterframework/parameters/Parameter.java

```diff
@@ -600,7 +605,7 @@ public Object getValue(ParameterValueList alreadyResolvedParameters, Message mes
 		if(result !=null && getType().requiresTypeConversion) {
 			result = getValueAsType(result, namespaceAware);
 		}
-		if (result !=null && result instanceof Number) {
+		if (result instanceof Number) {
```

Comment:
Als er nu een Number in de Message had gezeten, was het hier niet goed gegaan.

Reply:
Gelukkig stoppen wij geen nummers in message :D het gaat hier volgens mij met name om het resultaat dat uit `getValueAsType(result, namespaceAware);` komt. Die parst namelijk boolean/number/ed.